### DISCUSSION TITLE
Correct way of working with livewire and Metronic

### DISCUSSION THREAD

**Discussion by bashman** [livewire/livewire#5392] [1. Help]
Does anyone have any experience working with Metronic Admin Theme?... 

I am working with demo1, for laravel.
I have followed several tutorials to work with livewire.
Is it possible to integrate livewire with Metronic 8, is there any way to properly integrate livewire?

**Comment by joshhanley** [comment]
@bashman I've not used that, so can't be of much help. But I noticed it is a bootstrap based theme. Livewire and bootstrap can be made to work nicely together but it can take a fair bit of work depending on which parts of bootstrap the theme uses. If it's just css it should be fine, but if there is any interactive elements (using javascript to generate), you will have issues with Livewire overwriting the bootstrap generated html, and takes work to ensure that doesn't happen.

Hope this helps!
